README.md

```diff
@@ -14,11 +14,42 @@ Just pass a `loader` method which return a async module import:
 <Loadable loader={() => import('./AsyncComponent.svelte')} />
 ```
 
+Use `unloader` to prevent Loadable from caching the component which will cause it to call `loader` each time the component is used.
```

Comment:
```suggestion
Use `unloader` to prevent `Loadable` from caching the component which will cause it to call `loader` each time the component is used after being unmounted.
```